buildSrc/src/main/kotlin/JavaBasedProjectConventions.kt

```diff
@@ -30,28 +30,24 @@ fun Project.javaBasedProjectConventions() {
     }
 
     dependencies {
-        "api"(kotlin("stdlib", "1.5.32"))
```

Comment:
this whole file is a good candidate to become a convention plugin; then you can use the regular `api(...)` syntax and other dsls

Reply:
I'm not sure I can use the regular syntax since I won't have a Kotlin plugin applied in this file to generate those accessors - this is used both in Android and Kotlin JVM projects.

Reply:
ah ok